instrumentation/apache-httpclient/apache-httpclient-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/v5_0/ApacheHttpClientHelper.java

```diff
@@ -3,9 +3,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.javaagent.instrumentation.apachehttpclient.v5_0;
+package io.opentelemetry.javaagent.instrumentation.v5_0;
```

Comment:
i've had this Intellij refactoring problem before too 😄
```suggestion
package io.opentelemetry.javaagent.instrumentation.apachehttpclient.v5_0;
```

Reply:
done